Automattic/newspack-newsletters
# fix: update stylelint version

### All Submissions:






### Changes proposed in this Pull Request:

The update of the CI image for the `build` task from `circleci/php:7.2-node-browsers` to `circleci/php:7.4-node-browsers` requires an update to the `stylelint-config-wordpress`.

The `node-sass` dependency version from the current version of this package is not compatible with the recently updated build image.



### How to test the changes in this Pull Request:

1. On the master branch, run `circleci local execute --job build` and observe the error
2. Switch to this branch, run `npm ci` and `circleci local execute --job build` and observe that the task succeeds

### Other information:
